py4DSTEM/datacube/virtualdiffraction.py

```diff
@@ -184,22 +183,33 @@ def get_virtual_diffraction(
             qx_shift = x0_mean - x0
             qy_shift = y0_mean - y0
 
-            # ...for integer shifts
-            if not subpixel:
+            if subpixel is False:
```

Comment:
why change `not subpixel` to `subpixel is False`? I think the former should be preferred